src/livekit/useLiveKit.ts

```diff
@@ -137,22 +142,47 @@ export function useLiveKit(
     // and setting tracks to be enabled during this time causes errors.
     if (room !== undefined && connectionState === ConnectionState.Connected) {
       const participant = room.localParticipant;
-      if (participant.isMicrophoneEnabled !== muteStates.audio.enabled) {
-        participant
-          .setMicrophoneEnabled(muteStates.audio.enabled)
-          .catch((e) =>
-            logger.error("Failed to sync audio mute state with LiveKit", e)
-          );
-      }
-      if (participant.isCameraEnabled !== muteStates.video.enabled) {
-        participant
-          .setCameraEnabled(muteStates.video.enabled)
-          .catch((e) =>
-            logger.error("Failed to sync video mute state with LiveKit", e)
-          );
-      }
+
+      const syncMuteStateAudio = async () => {
+        if (
+          participant.isMicrophoneEnabled !== muteStates.audio.enabled &&
+          !audioMuteUpdating
+        ) {
+          setAudioMuteUpdating(true);
+          await participant
+            .setMicrophoneEnabled(muteStates.audio.enabled)
+            .catch((e) =>
```

Comment:
probably want this to be a try block too, to avoid mixing promises / await syntax